skimage/restoration/deconvolution.py

```diff
@@ -383,3 +383,96 @@ def richardson_lucy(image, psf, iterations=50, clip=True, filter_epsilon=None):
         im_deconv[im_deconv < -1] = -1
 
     return im_deconv
+
+
+def DNP_Gauss_freq(image, psf, we=0.01, clip=True):
+    """ Deconvolution using natural image priors
+        with a Gaussian prior, solved in the frequency domain
```

Comment:
```suggestion
def gaussian_natural_prior(image, psf, we=0.01, clip=True):
    """Deconvolution using Gaussian natural image priors.
```

(following PEP-8 and PEP-257)

I would also rename `we` to `smoothness_weight`.

Finally, I would like to see a bit of information in the argument description about what it does, rather than forcing users to go find a paper.

Reply:
done. thank you. ready for next commit

Reply:
The function name change to `gaussian_natural_prior` remains to be done (skimage functions should have [snake case](https://en.wikipedia.org/wiki/Snake_case) names)

Reply:
done in latest commit